packages/steiger-plugin-fsd/src/forbidden-imports/README.md

```diff
@@ -173,8 +203,6 @@ flowchart BT
 
 This is one of the main rules of Feature-Sliced Design, it ensures low coupling and predictability in refactoring.
 
-## Related Rules
-
 If you need more granular control, this rule's functionality is split into two separate rules that you can use independently:
 
 - `no-higher-level-imports`: Only checks for imports from higher layers (e.g., features importing from pages)
```

Comment:
suggestion: looks like this part should be removed now that it's been moved to the top